libsql-sqlite3/src/build.c

```diff
@@ -5628,19 +5628,30 @@ KeyInfo *sqlite3KeyInfoOfIndex(Parse *pParse, Index *pIdx){
   int nCol = pIdx->nColumn;
   int nKey = pIdx->nKeyCol;
   KeyInfo *pKey;
+  char* zIndexName = NULL, *zDbSName = NULL;
```

Comment:
Do we need those variables at all? Couldn't we just use pKey->zIndexName and pKey->zDbSName directly?